lib/govuk_navigation_helpers.rb

```diff
@@ -1,11 +1,22 @@
-require "govuk_navigation_helpers/version"
-require "govuk_navigation_helpers/breadcrumbs"
-require "govuk_navigation_helpers/related_items"
-require "govuk_navigation_helpers/taxon_breadcrumbs"
-require "govuk_navigation_helpers/taxonomy_sidebar"
-require "govuk_navigation_helpers/related_navigation_sidebar"
-require "govuk_navigation_helpers/rummager_taxonomy_sidebar_links"
-require "govuk_navigation_helpers/curated_taxonomy_sidebar_links"
+require 'active_support'
+require 'active_support/core_ext/hash/keys'
+require 'active_support/core_ext/object/blank'
+
+require 'govuk_ab_testing'
+
+require_relative "govuk_navigation_helpers/version"
```

Comment:
I've changes this so I can run the gem in `irb` by doing `irb -r govuk_navigation_helpers `